CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
   - Fixed add/create, delete/remove for resources
   - Added support to Enclosure group actions
   - Added support to Ethernet network `:bulk_create`
+  - Added support to Logical Enclosure actions
```

Comment:
Logical **enclosure** already had some actions already implemented before.

Could you please specify what changed? (and also fix the name to comply with the standard)